utilization.md

```diff
@@ -1,142 +1,198 @@
-|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `-kokkos-help`           | Print help message with the list of available options                                                                                               |
-| `-kokkos-threads=INT`    | Specify total number of threads or number of threads per NUMA region if used in conjunction with `--kokkos-numa` option                             |
-| `-kokkos-numa=INT`       | Specify number of NUMA regions used by process                                                                                                      |
-| `-device-id=INT`         | Specify device ID to be used by Kokkos                                                                                                              |
-| `-num-devices=INT[,INT]` | Specify the number of devices per node to be used when running MPI jobs; the optional second argument allows for an existing device to be ignored |
+| Execution space                     | Device backend | Host backend |
+|-------------------------------------|----------------|--------------|
+| `Kokkos::DefaultExecutionSpace`     | On device      | On host      |
+| `Kokkos::DefaultHostExecutionSpace` | On host        | On host      |
+| `Kokkos::Serial`                    | On host        | On host      |
 
-#### Struct arguments
+`Kokkos::Serial` is used for debug purpose only.
+
+<!--#ifndef PRINT-->
+<img title="Doc" alt="Doc" src="./images/doc_txt.svg" height="25"> https://kokkos.org/kokkos-core-wiki/API/core/execution_spaces.html
+<!--#endif-->
+
+### Memory spaces
+
+<!--#ifndef PRINT-->
+<img title="Doc" alt="Doc" src="./images/doc_txt.svg" height="25"> https://kokkos.org/kokkos-core-wiki/API/core/memory_spaces.html
+<!--#endif-->
+
+#### Generic memory spaces
+
+| Memory space                                      | Device backend | Host backend |
+|---------------------------------------------------|----------------|--------------|
+| `Kokkos::DefaultExecutionSpace::memory_space`     | On device      | On host      |
+| `Kokkos::DefaultHostExecutionSpace::memory_space` | On host        | On host      |
+
+#### Specific memory spaces
+
+| Memory space                 | Description                                                                  |
+|------------------------------|------------------------------------------------------------------------------|
+| `Kokkos::HostSpace`          | Accessible by the host but not directly by the device                        |
+| `Kokkos::SharedSpace`        | Accessible by the host and the device; synchronization managed by the driver |
+| `Kokkos::ScratchMemorySpace` | Accessible by the team that created it and nothing else                      |
```

Comment:
No thread level scratch? https://kokkos.org/kokkos-core-wiki/API/core/policies/TeamHandleConcept.html?highlight=scratch#_CPPv4NK17TeamHandleConcept14thread_scratchEi

Reply:
I'll modify it to include the thread as well.